tests/common.go

```diff
@@ -2133,15 +2160,18 @@ func SetSourceKubeConfig() error {
 	if err != nil {
 		return err
 	}
-	SetClusterContext(sourceClusterConfigPath)
-	return nil
+	err = SetClusterContext(sourceClusterConfigPath)
+	return err
 }
 
 // SetDestinationKubeConfig sets current context to the kubeconfig passed as destination to the torpedo test
-func SetDestinationKubeConfig() {
+func SetDestinationKubeConfig() error {
```

Comment:
Please refer to PR #1225. How about handling errors where SetDestinationKubeConfig is used?

Reply:
Correct. SetDestinationKubeConfig is used in many test cases . Please handle them all

Reply:
Thanks for reminding me.
Making changes...

Reply:
✅